samples/dotnet/kernel-syntax-examples/Example12_Planning.cs

```diff
@@ -75,7 +75,7 @@ private static async Task EmailSamplesAsync()
         Console.WriteLine(originalPlan.Variables.ToPlan().PlanString);
 
         var executionResults = originalPlan;
-        executionResults.Variables.Update(
+        _ = executionResults.Variables.Update(
```

Comment:
nit: I understand that discarding can be useful to indicate that you want to ignore the variable, but taking into account that no variable was declared should we add it the examples? It makes the examples a little bit "noisy".